networks/amity/config.xml

```diff
@@ -66,9 +66,9 @@
         <state-storage>FULL</state-storage>
         <!--Database implementation used to store data; supported options: leveldb, h2, rocksdb.-->
         <!--Caution: changing implementation requires re-syncing from genesis!-->
-        <vendor>leveldb</vendor>
+        <vendor>rocksdb</vendor>
         <!--Boolean value. Enable/disable database compression to trade storage space for execution time.-->
-        <enable_db_compression>false</enable_db_compression>
+        <enable_db_compression>true</enable_db_compression>

```

Comment:
For LevelDB the compression made performance fairly bad. Do we have experimental data to show that it's better to enable compression with RocksDB?

Reply:
I can do the benchmark. But bast on my understanding, Enable the data compression will increase CPU usage but significantly reduce the data size been wrote into the disk, especially the compacting happens. Also I doubt the leveldb library might use the old version of the native levelDB, So the compression performance is not good. for the benchmark I did before. You can look at the AKI-550. 
Also,you can look at the reference link for the details of the rocksdb data compression
https://github.com/facebook/rocksdb/wiki/Compression

Reply:
Also, the default setting in rust kernel also enable the data compression.
https://github.com/aionnetwork/aionr/blob/master/db/core/src/dbconfigs.rs#L82-L100